Nodejs/Product/TestAdapter/TestAdapter.csproj

```diff
@@ -9,8 +10,12 @@
     <OldToolsVersion>4.0</OldToolsVersion>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <GeneratePkgDefFile>false</GeneratePkgDefFile>
```

Comment:
Does this not generate the pkgdef? If not, what does? You added an attribute above for use in pkgdef generation.